### ISSUE TITLE
Load and Index Archived MURs

### ISSUE THREAD

**Issue by porta-antiporta** [18F/fec-eregs#220]
Our universal search should be able to search through both current and archived MURs.  #49 loads up current MURs.  This story is focused around Archived MURs.  There are 4446 total MURs

Archived MURs have the following distinction from Current MURs
- They are not loaded up in an Oracle DB; hence they don't have the same metadata.
    - Meta data is manually created by the FEC and loaded up in static website [tables](http://www.fec.gov/MUR/)
        - Open Date
        - Closed Date
        - Claimants/Respondents
        - Subject
        - Citation
- A single PDF scan is used to encase all the documents within the MUR itself

Thus as we look to load and index archived MURs we should:
- [ ] Ensure we scrape the table meta data into our ES index
- [ ] Evaluate how weighting within results that include current MURs should be considered.  Given a phrase would be searched across "all the docs" when it comes to an archived MUR
    - [ ] This may be OK if we want to prioritize current MURs in search results (i.e. more recent is more relevant).  But lets make it an explicit decision/discussion.

**Comment by porta-antiporta** [comment]
Commenting on the thread starting here: https://18f.slack.com/archives/fec-legal/p1472154913001317

TL:DR; There are aspects in the UI that we will not be able to support for archived MURs because that is not available.

## For the search results page
https://gsa.invisionapp.com/share/AR7U3Y662#/167711268_Enforcement_-_Search
- Committee
- Disposition
- # Documents
- Discuss with @jenniferthibault whether we should just leave those blank or should have a "Not Available"

## For the MUR canonical page
https://gsa.invisionapp.com/share/AR7U3Y662#/167711271_MUR_Canonical
- Omit Committee `Canseco for Congress`
- Omit Summary
- Omit Disposition
- Omit Participants
- Just have a single document listed

## Future faceting
- Allow for a "Not available" selection so that archived MURs can be included in the resultant set?

**Comment by anthonygarvan** [comment]
Thanks @porta-antiporta! We can include Participants in the canonical page, that info is in the meta data.

**Comment by jenniferthibault** [comment]
For the search page, here's my first stab at how we could integrate archived MURs to the results stream, and give people hints about why the available data is different:

<img width="772" alt="screen shot 2016-08-26 at 9 46 50 am" src="https://cloud.githubusercontent.com/assets/11636908/18007425/0b829134-6b72-11e6-997c-8100e0913137.png">

Including the `subject` field is helpful to give a hint about what the MUR is about, but it does make the row quite a bit longer.

Thoughts?
If something isn't clear or I've done something counter-intuitive, there's a good chance it's because I don't fully understand what's happening, so let me know if it's seems off :)

Is this all you need, or do you also need some visual thinking about what happens on a canonical page? Trying to get up to speed, sorry!

**Comment by anthonygarvan** [comment]
Looks great @jenniferthibault ! Yes I think some clear guidance on the canonical page would be helpful as well, otherwise we'll be winging it a bit.

**Comment by jenniferthibault** [comment]
Here's the updated search line item for an archived MUR 
(changes: removes the PDF size, and replaces the `subject` with the snippet)

<img width="713" alt="screen shot 2016-08-26 at 11 11 06 am" src="https://cloud.githubusercontent.com/assets/11636908/18010308/f0e52448-6b7d-11e6-9f9d-0d218447929b.png">


And here's how we could make a canonical page for archived MURs:
Instead removing the Summary field, I think we should keep it, but use the metadata fields from the archived data under it. This way it still feels like it's based off the same template as a current MUR.

![mur-archived-canonical](https://cloud.githubusercontent.com/assets/11636908/18010357/2a2f433c-6b7e-11e6-8bcd-4d3a325e8bc8.png)

**Comment by jenniferthibault** [comment]
I had missed @anthonygarvan 's comment about the Participants before, so just updated the image above to add it in.

**Comment by porta-antiporta** [comment]
@jenniferthibault The AO canonical page mockup has a different pattern to display citations.  https://gsa.invisionapp.com/share/AR7U3Y662#/167711255_AO_Index_-2-.  We haven't implemented citations on AO canonical pages but should they be consistent (and which pattern should we use?)

<img width="1280" alt="screen shot 2016-08-26 at 11 32 58 am" src="https://cloud.githubusercontent.com/assets/16668007/18012851/deb13ebc-6b80-11e6-8d60-4698ca2c331a.png">

**Comment by jenniferthibault** [comment]
@porta-antiporta good eye, I was working too fast this morning. Yes, I'd like to use the side panel for citations consistently.

The difference here is that the MUR canonical page uses the side-nav, and the AO canonical page does not. We haven't combined the side nav with a resources panel anywhere before (which both creep into the main content area from opposite sides). I think it works out ok in this case because the `Subject` field in the archived MURs are pretty short. Game to give it a go!


I'm not fluent in legal abbreviations to know for sure, but is the second citation in this example referencing a statute? Or is it also a regulation? The pattern should use a title for each group type, if they are different.

![mur-archived-canonical copy](https://cloud.githubusercontent.com/assets/11636908/18015071/0cd0b9ae-6b94-11e6-8965-3a2bbfc3178e.png)

**Comment by porta-antiporta** [comment]
@jenniferthibault yes that's correct.
CFR - Code of Federal Regulations (Regulations
USC - United States Code (Statute)

Re: side nav.  I think that can be very useful for not just archived MURs, but current MURs ( and maybe even AOs as well) given that the list of assets can be quite long.  We don't have the Summary for

**Comment by jenniferthibault** [comment]
@porta-antiporta your comment cut off, what were you going to say about the Summary?


And the side nav pattern is one I grabbed here because it was already in the current MUR template, and I think the MUR pages should be as consistent as possible between current & archived. 

It's definitely a helpful tool when a page has a lot of sections, but AO pages only have two at the moment ([Documents and Entities](https://beta.fec.gov/data/legal/advisory-opinions/1996-15/)) so I'm not sure we need it just yet. 



 <img width="513" alt="screen shot 2016-08-26 at 2 33 53 pm" src="https://cloud.githubusercontent.com/assets/11636908/18016417/43caa2f2-6b9a-11e6-8a99-52c4ef71d2ac.png">

**Comment by porta-antiporta** [comment]
Ah I was saying that we haven't implemented Summary on AOs yet as they need to be scraped off the FEC Record.  But :+1: on not having the side nave on AOs until we have a demonstrated need for it.

**Comment by porta-antiporta** [comment]
We have the MUR names after all from a the case management system at the FEC.
https://18f.slack.com/archives/fec-legal/p1472565965000033

**Comment by jenniferthibault** [comment]
Hey @anthonygarvan I moved the design & styling work into a new issue ( #244 ) in hopes that this issue could continue forward without being blocked by the design work.

**Comment by porta-antiporta** [comment]
@anthonygarvan Let's say a user enters a keyword that matches the `Subject` or `Citation` that is part of a particular MURs metadata:
* Will it return the case as a result?
* If so, in what priority is the result ranked compared to text matches?
* What snippet (if any) is returned along with that result?

Note: A perfectly valid answer is: "@porta-antiporta, waaay out of scope for this story and you need to let me know what  the desired behavior is from the user in that new story"

**Comment by jenniferthibault** [comment]
@anthonygarvan the design questions from #244 are wrapped up and I've updated the following files with the final designs:
- desktop InVision: 
  - [current MUR](https://gsa.invisionapp.com/share/S58HMFKMB#/185960395_MUR_Canonical)
  - [archived MUR](https://gsa.invisionapp.com/share/S58HMFKMB#/185961446_MUR-Archived-Canonical-Desktop) 
- mobile InVision:
  - [current MUR](https://gsa.invisionapp.com/share/MA8ID5U6Z#/186537552_MUR-Canonical-Mobile)
  - [archived MUR](https://gsa.invisionapp.com/share/MA8ID5U6Z#/186537551_MUR-Archived-Canonical-Mobile)

**Comment by anthonygarvan** [comment]
@porta-antiporta @adborden answers:

- Will it return the case as a result? Yes.
- If so, in what priority is the result ranked compared to text matches? Same weight as the text.
- What snippet (if any) is returned along with that result? In the old behavior, just text, name and number were highlighted, so fields like citation or other non-standard fields would not be included.
